packages/theme/oclif.manifest.json

```diff
@@ -1315,6 +1315,67 @@
         }
       },
       "args": {}
+    },
+    "theme:update:run": {
+      "id": "theme:update:run",
+      "description": "Run the theme update process.",
+      "strict": true,
+      "pluginName": "@shopify/theme",
+      "pluginAlias": "@shopify/theme",
+      "pluginType": "core",
+      "aliases": [],
+      "flags": {
+        "no-color": {
+          "name": "no-color",
+          "type": "boolean",
+          "description": "Disable color output.",
+          "hidden": false,
+          "allowNo": false
+        },
+        "verbose": {
+          "name": "verbose",
+          "type": "boolean",
+          "description": "Increase the verbosity of the logs.",
+          "hidden": false,
+          "allowNo": false
+        },
+        "path": {
+          "name": "path",
+          "type": "option",
+          "description": "The path to your theme directory.",
+          "multiple": false,
+          "default": "."
+        },
+        "store": {
+          "name": "store",
+          "type": "option",
+          "char": "s",
+          "description": "Store URL. It can be the store prefix (johns-apparel) or the full myshopify.com URL (johns-apparel.myshopify.com, https://johns-apparel.myshopify.com).",
+          "multiple": false
+        },
+        "script": {
+          "name": "script",
+          "type": "option",
+          "description": "The path to the 'update_extension.json' script.",
+          "multiple": false,
+          "default": "./update_extension.json"
```

Comment:
Could this be problematic if they have an `./update_extension.json` in their directory but it was created for a different theme but they want to test an update for a different theme that doesn't need any `update_extension`. The CLI would be picking up this script to run with any update command incorrectly 🤔